dnf-behave-tests/dnf/history-list-dnf5.feature

```diff
@@ -0,0 +1,147 @@
+# This tests are somehow redundat for dnf4. the features
+# are tested for history command. Example:
+# history list last-1 has the same behavior as history last-1
+# in dnf4
+#
+# This feature file can be merged with history-list.feature
+# in the future if the behavior of history command will change
+# assuming list by default
+@dnf5
+Feature: history list list
```

Comment:
I don't like the duplication of tests in this file. The reason here is dnf4 supports omitting the subcommand and defaults to `list`, and there's a bunch of tests like `dnf history last`. I think it would be much better to rewrite the tests to include the subcommand: `dnf history list last` and then, only for dnf4 specifically test that the shorter version works. You should only need a couple of tests for that, and the rest of the tests should be shared if possible.

The way it is now, the dnf5 tests are split between the old file and this new one, and it's hard to make sense of it and see what is tested where and for which version, etc. It adds chaos which will be hard and time-consuming to maintain.